modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/AbstractZoneReplicationTest.java

```diff
@@ -78,11 +81,11 @@
 abstract class AbstractZoneReplicationTest extends IgniteAbstractTest {
     private static final int BASE_PORT = 20_000;
 
-    protected static final String TEST_ZONE_NAME = "TEST_ZONE";
+    static final String TEST_ZONE_NAME = "TEST_ZONE";
```

Comment:
Let's remove this class and use `ItAbstractColocationTest` instead. It seems to me, that there is no value in having two base classes that do the same.

Reply:
I switched the test class I added in this PR to `ItAbstractColocationTest`, but I left the other base class for now as it seems to be an unrelated task to remove it; I filed an issue about it https://issues.apache.org/jira/browse/IGNITE-24649